cloud-gateway-service/src/main/java/org/zowe/apiml/cloudgatewayservice/config/ConnectionsConfig.java

```diff
@@ -195,12 +210,56 @@ public CloudEurekaClient primaryEurekaClient(ApplicationInfoManager manager, Eur
         AbstractDiscoveryClientOptionalArgs<?> args = new MutableDiscoveryClientOptionalArgs();
         args.setEurekaJerseyClient(eurekaJerseyClient);
 
-        CloudEurekaClient cloudEurekaClient = new CloudEurekaClient(appManager, config, args,
-            this.context);
+        final CloudEurekaClient cloudEurekaClient = new CloudEurekaClient(appManager, config, args, this.context);
         cloudEurekaClient.registerHealthCheck(healthCheckHandler);
         return cloudEurekaClient;
     }
 
+    @Bean
+    public List<AdditionalRegistration> additionalRegistration(StandardEnvironment environment) {
+        List<AdditionalRegistration> additionalRegistrations = new AdditionalRegistrationParser().extractAdditionalRegistrations(System.getenv());
+        log.debug("Parsed {} additional regs, \t first: {}", additionalRegistrations.size(), additionalRegistrations.stream().findFirst().orElse(null));
```

Comment:
better to not use abbreviation but rather use `registrations`

Reply:
It is similar to what we have in the `GATEWAY` -  `apiml.service.additionalRegistration` yml configuration which get translated in to the `additionalRegistraion` Bean.

Reply:
It is similar to what we have in the `GATEWAY` -  `apiml.service.additionalRegistration` yml configuration which get translated in to the `additionalRegistraion` Bean.